.github/ISSUE_TEMPLATE/new_language.md

```diff
@@ -0,0 +1,29 @@
+---
+name: New language
+about: Request support for a new language
+title: ''
+labels: Add Language, Good First Issue
+assignees: ''
+---
+
+## Language Name
+
+
+## URL of example repository
```

Comment:
Suggested addition:

```markdown
# Can't find a syntax highlighting grammar?

* [ ] Request @Alhadis to write one
	* [ ] Request it _now_, not in sixteen months' time at 4am
```

It's only a matter of time before [`vendors/README.md`](https://github.com/github/linguist/blob/master/vendor/README.md) gets replaced with a link to my profile saying _`Go bug this guy`_.

Reply:
In all seriousness, we could do with a `URL of syntax highlighting grammar`, which eliminates the need to shop around for an existing one.

Reply:
😆. Seriously, really want this kind of burden embedded in the template?

Reply:
Whoops. My browser didn't update 🤦